#represents the position of every crazyflie
#the index corresponding with the crazyflie's ID tells you its x, y, and z location
#if put into the obvious list


int32[] x
int32[] y
int32[] z
